### PR TITLE
increase the size of create payload to 200k

### PR DESCRIPTION
**Pull request by johnduffell**
This PR allows users to send double the amount of body size to the create endpoint.  From 100k to now 200k for this endpoint.

This means that users can spend more time editing their details and building up debug info before it would get a 413 error from the endpoint.

Although the documentation for play-circe is not too detailed, it's clear how to override the max length https://github.com/jilen/play-circe/blob/da0959ea58c8eb42b92e9c1bea88c523a2f56f74/src/main/scala/play/api/libs/circe/Circe.scala#L42

the default global for play is set here: https://www.playframework.com/documentation/3.0.0/api/scala/play/api/mvc/PlayBodyParsers.html#DefaultMaxTextLength:Long

This PR also includes a little refactoring.

Tested locally and created a Guardian Weekly successfully.

### CODE REVIEW

File: support-frontend/app/controllers/CreateSubscriptionController.scala [hunk 1]
```diff
@@ -26,7 +26,7 @@ import play.api.libs.circe.Circe
 import play.api.mvc._
 import services.AsyncAuthenticationService.IdentityIdAndEmail
 import services.stepfunctions.{CreateSupportWorkersRequest, StatusResponse, SupportWorkersClient}
-import services.{IdentityService, RecaptchaResponse, RecaptchaService, StripeSetupIntentService, TestUserService}
```
Comment (johnduffell) [inline support-frontend/app/controllers/CreateSubscriptionController.scala:29]: unused

File: support-frontend/app/controllers/CreateSubscriptionController.scala [hunk 2]
```diff
@@ -36,11 +36,9 @@ import scala.util.chaining._
 
 object CreateSubscriptionController {
 
-  sealed abstract class CreateSubscriptionError(message: String)
-  case class ServerError(message: String) extends CreateSubscriptionError(message)
-  case class RequestValidationError(message: String) extends CreateSubscriptionError(message)
-
-  type ApiResponseOrError[RES] = EitherT[Future, CreateSubscriptionError, RES]
```
Comment (johnduffell) [inline support-frontend/app/controllers/CreateSubscriptionController.scala:43]: unused

File: support-frontend/app/controllers/CreateSubscriptionController.scala [hunk 3]
```diff
@@ -36,11 +36,9 @@ import scala.util.chaining._
...
+  private sealed abstract class CreateSubscriptionError
+  private case class ServerError(message: String) extends CreateSubscriptionError
+  private case class RequestValidationError(message: String) extends CreateSubscriptionError
```
Comment (johnduffell) [inline support-frontend/app/controllers/CreateSubscriptionController.scala:41]: intellij told me these could be private

File: support-frontend/app/controllers/CreateSubscriptionController.scala [hunk 4]
```diff
@@ -64,9 +62,17 @@ class CreateSubscriptionController(
...
+    circe.json(maxCreateJsonLength).validate { json =>
+      val decoderResult = json.as[CreateSupportWorkersRequest]
+      decoderResult.leftMap(error => Results.BadRequest(error.show))
+    }
+  }
+
   def create: EssentialAction =
     LoggingAndAlarmOnFailure {
-      MaybeAuthenticatedActionOnFormSubmission.async(circe.json[CreateSupportWorkersRequest]) { implicit request =>
+      MaybeAuthenticatedActionOnFormSubmission.async(createRequestBodyParser) { implicit request =>
```
Comment (johnduffell) [inline support-frontend/app/controllers/CreateSubscriptionController.scala:75]: this is the necessary change, I've extracted it to make it clearer.

**Review by tomrf1 — APPROVED** [review]
great

**Pull request merged**